TypeCobol/Compiler/CupParser/NodeBuilder/ProgramClassBuilder.cs

```diff
@@ -72,7 +72,15 @@ public SymbolTable CustomSymbols
                 }
                 // TODO#249: use a COPY for these
                 foreach (var type in DataType.BuiltInCustomTypes)
-                    TableOfIntrisic.AddType(DataType.CreateBuiltIn(type)); //Add default TypeCobol types BOOLEAN and DATE
+                {
+                    var createdType = DataType.CreateBuiltIn(type);
+                    TableOfIntrisic.AddType(createdType); //Add default TypeCobol types BOOLEAN and DATE
+                    //Add children to dictionary in Intrinsic symbol table
+                    if (createdType.Children.Count != 0)
```

Comment:
Why count children here?
If a Type has no children, method AddTypeDataDefinition handle this case + it also have a behavior when there is no children.